metrics/ap_pipe.yaml

```diff
@@ -22,3 +22,26 @@ ApPipeMemory: &Memory
         - task
         - monitoring
 
+ApFakesCompletenessMag20t22:
+    description: Completeness of recovered fakes in the magnitudes range 20-22.
+    unit: '%'
+    tags:
+        - ap_verify
+        - task
+        - monitoring
+
+ApFakesCompletenessMag22t24:
+    description: Completeness of recovered fakes in the magnitudes range 22-24.
+    unit: '%'
+    tags:
+        - ap_verify         
+        - task
+        - monitoring
+
```

Comment:
Since all these metrics have identical units and tags, I'd recommend using `*`/`<<` to reduce the boilerplate. You can find an example in `pipe_tasks.yaml`.